bom/pom.xml

```diff
@@ -30,6 +30,7 @@
 
   <properties>
     <artemis.version>2.1.0</artemis.version>
+    <artemis.image.name>enmasseproject/activemq-artemis:2.2.0-1</artemis.image.name>
```

Comment:
shouldn't we remove `artemis.version` in this case? Or at least use the same version (2.2.0)?